### ISSUE TITLE
[Question] Is there a way for CZ to ignore the filesystem in the image?

### ISSUE THREAD

**Issue by rado84-github** [stevenshiau/clonezilla#91]
Is there a way for CZ to ignore the filesystem in the image and pour the files in an XFS partition? The point is that the next time I make a backup image, the new filesystem in the image to be XFS, not ext4.

**Comment by JonnyTech** [comment]
Have you considered converting with [fstransform](https://github.com/cosmos72/fstransform)?

**Comment by stevenshiau** [comment]
No, the file system engine of Clonezilla, Partclone, is actually block-based. In other words, actually it does not know files on a file system. It just realizes the used blocks on the file system.
I suggest you can use Clonezilla to backup the file system, and use fstransform to do this conversion.

Steven
